### PR TITLE
add logging to `x.size()` case in `peephole.cpp`

### PR DESCRIPTION
**Pull request by standbyme**
Add `GRAPH_UPDATE` for `x.size()` in Peephole Optimize

### CODE REVIEW

File: torch/csrc/jit/passes/peephole.cpp [hunk 1]
```diff
@@ -228,6 +228,10 @@ void PeepholeOptimizeImpl(Block* block, bool addmm_fusion_enabled) {
           WithInsertPoint guard(node);
           IValue ival(sizes);
           auto const_sizes_val = node->owningGraph()->insertConstant(ival);
+          GRAPH_UPDATE(
+              *node,
+              " (x.size()) is replaced with ",
+              const_sizes_val->debugName());
```
Comment (Krovatkin) [inline torch/csrc/jit/passes/peephole.cpp]: could you please use `getHeader(node)` instead of `*node`. Otherwise, looks good!

**Review by Krovatkin — CHANGES_REQUESTED** [review]
could you please change `*node` to `getHeader(node)`? Thanks!

File: torch/csrc/jit/passes/peephole.cpp (hunk 1 above)
Comment (standbyme) [reply torch/csrc/jit/passes/peephole.cpp]: Done~

Should I replace all the previous style `*node` with `getHeader(node)`? 😀

**Comment by kostmo** [comment]
## :pill: CircleCI build failures summary and remediations
As of commit cf385ae1:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=cf385ae1392a8da86a817f0b8b9725e282dab676).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4128075) pytorch_xla_linux_xenial_py3_6_clang7_build (1/1)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=80143504) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=80143504))
<details>
<summary>
<code>Jan 04 01:52:55 caused by: Connection refused (os error 111)</code>
</summary>

```
Jan 04 01:52:55 +++ eval 'extract_trap_cmd ' 
Jan 04 01:52:55 ++++ extract_trap_cmd 
Jan 04 01:52:55 ++++ printf '%s\n' '' 
Jan 04 01:52:55 +++ printf '%s\n' cleanup 
Jan 04 01:52:55 ++ trap -- ' 
Jan 04 01:52:55 cleanup' EXIT 
Jan 04 01:52:55 ++ which sccache 
Jan 04 01:52:55 ++ sccache --stop-server 
Jan 04 01:52:55 Stopping sccache server... 
Jan 04 01:52:55 error: couldn't connect to server 
Jan 04 01:52:55 caused by: Connection refused (os error 111) 
Jan 04 01:52:55 ++ true 
Jan 04 01:52:55 ++ rm /var/lib/jenkins/sccache_error.log 
Jan 04 01:52:55 rm: cannot remove '/var/lib/jenkins/sccache_error.log': No such file or directory 
Jan 04 01:52:55 ++ true 
Jan 04 01:52:55 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Jan 04 01:52:55 ++ SCCACHE_IDLE_TIMEOUT=1200 
Jan 04 01:52:55 ++ RUST_LOG=sccache::server=error 
Jan 04 01:52:55 ++ sccache --start-server 
Jan 04 01:52:55 Starting sccache server... 
Jan 04 01:52:55 ++ sccache --zero-stats 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

**Comment by zou3519** [comment]
@Krovatkin how does this look?

**Review by Krovatkin — APPROVED** [review]
:shipit:

File: torch/csrc/jit/passes/peephole.cpp (hunk 1 above)
Comment (Krovatkin) [reply torch/csrc/jit/passes/peephole.cpp]: sure, that would be great!

**Comment by Krovatkin** [comment]
@zou3519 sorry for the delay in merging your PR.
I'm trying to merge this PR in first, 
https://github.com/pytorch/pytorch/pull/28455
and then I'll try to merge in yours.
Thanks!

**Comment by Krovatkin** [comment]
@standbyme could you please rebase yr PR now and I'll merge it in! Thanks! Sorry for the delay!

**Comment by standbyme** [comment]
I'm sorry I can't finish this soon. My computer was in the company, and I couldn't get back to work any time soon because of the pneumonia outbreak in China.😷
________________________________

**Comment by Krovatkin** [comment]
@standbyme ofc, this isn't urgent at all! stay safe!

**Comment by standbyme** [comment]
@Krovatkin Sorry for delay long time. I fix this PR on #34865

**Pull request closed without merging**